src/store/systems.js

```diff
@@ -0,0 +1,21 @@
+const systems = [
```

Comment:
This file could use a documentation comment explaining what it is and why it exists.